.github/workflows/checks-sealights.yaml

```diff
@@ -0,0 +1,184 @@
+# Copyright The Enterprise Contract Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+---
+name: Sealights
+
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened, labeled, unlabeled]
```

Comment:
I can it's different to the similar triggering config in .github/workflows/checks.yaml . Should we make them the same?

Reply:
Why include labeled & unlabeled here?

Reply:
Good catch we're not using labels in this way so I should remove labeled , unlabeled.

About doing this the same for every workflow. It could be good for consistency but its not needed IMO.
- I'm just learning about most of this today but the reason we are doing it here is that this workflow relies on a secret and so once this workflow is merged we'll be able to use this in future PRs since we have the pull_request_target here instead of pull_request. It means we'll use the workflow and secret from the upstream repo not the incoming PR. This prevents contributors from editing the workflow to cat the secret or something.

So for the other workflows if they do not require a secret to be used then using pull_request as they already do should be fine.